playwright/jest-setup.js

```diff
@@ -1,15 +1,20 @@
-const { getTestRunnerConfig, setPreRender, setPostRender, setupPage } = require('../dist');
+const { getTestRunnerConfig, setPreVisit, setPostVisit, setupPage } = require('../dist');
 
 const testRunnerConfig = getTestRunnerConfig(process.env.STORYBOOK_CONFIG_DIR);
 if (testRunnerConfig) {
+  // hooks set up
   if (testRunnerConfig.setup) {
     testRunnerConfig.setup();
   }
-  if (testRunnerConfig.preRender) {
-    setPreRender(testRunnerConfig.preRender);
+
+  const preVisitFn = testRunnerConfig.preRender || testRunnerConfig.preVisit;
```

Comment:
What if both are set? Why does preRender take priority? Should it error?

Reply:
Good catch! the order was supposed to be the other way around.